main.py

```diff
@@ -77,24 +114,35 @@ def resend_stored_flows(self, event):
             self.resent_flows.add(dpid)
             log.info(f'Flows resent to Switch {dpid}')
 
-    def is_ignored(self, flow):
+    @staticmethod
+    def in_range(field, exceptions):
```

Comment:
Maybe a better name would be 'is_listed' or 'is_exception'.

Reply:
Applied on https://github.com/kytos/flow_manager/pull/112/commits/74a7cc9910290e7128fb63ed7f9b5999b9564733 commit.